src/App.js

```diff
@@ -1,11 +1,23 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import AddTooForm from './AddTodoForm';
 import TodoList from './TodoList';
 
+function useSemiPersistentState() {
+  const [todoList, setTodoList] = useState(JSON.parse(localStorage.getItem("savedTodoList") || "[]"));
```

Comment:
This is working but this can lead to some bugs down the road. 

What data are we passing here? We have the local storage item and a string being passed to `getItem`. You have the right idea but consider only passing `savedTodoList` to `useState` or an array. Instead of `savedTodoList` or a string to `getItem`